src/production/urls.py

```diff
@@ -17,6 +17,10 @@
         views.preview_galley,
         name='production_preview_galley'
         ),
+    url(r'^(?P<article_id>\d+)/preview/(?P<galley_id>\d+)/(?P<file_name>.*)$',
+        views.preview_figure,
+        name='preview_figure'
```

Comment:
Name clashes with a url in `proofing.urls`

Also, should we add a link to this in the templates?

Reply:
I've updated that name, this url is for the production_preview_galley page to use when rendering HTML/XML. There is already a download link for galley images.